pages/common/aws-ecr.md

```diff
@@ -0,0 +1,36 @@
+# aws ecr
+
+> Push, pull, and manage container images.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ecr/index.html>.
+
+- Authenticate Docker with your default registry (username is AWS):
+
+`aws ecr get-login-password --region {{region}} | {{docker login}} --username AWS --password-stdin {{aws_account_id}}.dkr.ecr.{{region}}.amazonaws.com`
+
+- Create a repository:
+
+`aws ecr create-repository --repository-name {{repo_name}} --image-scanning-configuration scanOnPush={{true|false}} --region {{region}}`
+
+- Tag a local image for ECR:
+
+`docker tag {{container_name}}:{{tag}} {{aws_account_id}}.dkr.ecr.{{region}}.amazonaws.com/{{container_name}}:{{tag}}`
+
+- Push an image to a repository:
+
+`docker push {{aws_account_id}}.dkr.ecr.{{region}}.amazonaws.com/{{container_name}}:{{tag}}`
+
+- Pull an image from a repository:
+
+`docker pull {{aws_account_id}}.dkr.ecr.{{region}}.amazonaws.com/{{container_name}}:{{tag}}`
+
+- Delete an image from a repository:
+
+`aws ecr batch-delete-image  --repository-name {{repo_name}} --image-ids imageTag={{latest}}`
```

Comment:
If you always have to use `latest`, it should be
```suggestion
`aws ecr batch-delete-image  --repository-name {{repo_name}} --image-ids imageTag=latest`
```

Reply:
You don't have to — `latest` is the default, implied tag name for all Docker images.  The tag can be anything matching `[a-zA-Z0-9_.-]+` (i.e. numbers, letters, dashes, dots & underscores)

Reply:
```suggestion
`aws ecr batch-delete-image  --repository-name {{repository}} --image-ids imageTag={{latest}}`
```

Reply:
My suggestion overlaps with the other suggestion, deal with that one first

Reply:
I agree with @bl-ue. You can use an arbitrary image tag.